inc/saiport.h

```diff
@@ -369,6 +615,20 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_QOS_SCHEDULER_GROUP_LIST,
 
+    /**
+     * @brief The sum of the headroom size of the ingress priority groups belonging to this port
+     *        should not exceed the SAI_PORT_ATTR_QOS_MAXIMUM_HEADROOM_SIZE value.
+     *
+     * This attribute is applicable only for per-port, per-PG headroom model
+     * (which means SAI_BUFFER_POOL_ATTR_XOFF_SIZE is zero)
+     *
+     * For the platforms which don't have this limitation, 0 should be returned.
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_QOS_MAXIMUM_HEADROOM_SIZE,
```

Comment:
same here, you would need to number those enums explicitly or add at the end

Reply:
It is not my change. It is because of merge conflict it is automatically taking changes.